build-kong.sh

```diff
@@ -71,6 +71,10 @@ pushd /kong
   cp kong.conf.default /tmp/build/usr/local/lib/luarocks/rock*/kong/$ROCKSPEC_VERSION/
   cp kong.conf.default /tmp/build/etc/kong/kong.conf.default
   cp kong/pluginsocket.proto /tmp/build/usr/local/kong/lib
+
+  curl -fsSLo /tmp/protoc.zip https://github.com/protocolbuffers/protobuf/releases/download/v3.19.0/protoc-3.19.0-linux-x86_64.zip
```

Comment:
@javierguerragiraldez unsure about the version here. An idea is to pick it from the respective go-pdk version's `go.mod` file; what do you think?